### ISSUE TITLE
Problem with @Builder annotation and mixed variable types

### ISSUE THREAD

**Issue by mtrevisan** [rzwitserloot/lombok#1551]
I have this piece of code (lombok 1.16.18)

```
@Builder
@Getter
public class ParserOptions{

	@Builder.Default
	private final String commentMarker = "# ";

	@Builder.Default
	private final int maxRecursionLevels = 50;

	private final boolean partialParsing;

}
```

Netbeans 8.2 gives me the error "bad type in conditional expression, String cannot be converted to int".
One of the things I've tried was switching the first two variables. In this case the error changes into "int cannot be converted to String".
If both the first two variables were Strings or ints everything works fine.
It is my fault?

**Comment by rspilker** [comment]
Duplicate of #1547, which has just been fixed and released in 1.16.20.

**Comment by mtrevisan** [comment]
Ok, thank you. I'll wait it to be present in the maven repository.
